deploy-board/deploy_board/webapp/host_views.py

```diff
@@ -103,6 +103,13 @@ def get_host_details(host_id):
      'Launch Time': launch_time,
      'AMI Id': ami_id,
     }
+    if IS_PINTEREST:
+        from brood.client import Brood
+        cmdb = Brood()
+        host_ip = cmdb.get_query(query="id:" + ami_id,
```

Comment:
You can just use the above instance object. It will have the config.internal_address you are looking for . It is queried from cmdb.